src/views/FindProductInventory.vue

```diff
@@ -140,66 +140,66 @@
 
           <hr />
 
-          <div class="product" @click="() => router.push('/product-inventory')">
+          <div class="product" v-for="product in products" :key="product.groupValue" @click="() => router.push('/product-inventory')">
             <div class="desktop-only">
-              <Image src="https://cdn.shopify.com/s/files/1/0069/7384/9727/products/test-track.jpg?v=1626255137" />
+              <Image :src="getProduct(product.productId).mainImageUrl" />
             </div>
 
             <div>
               <section class="section-header">
                 <div class="primary-info">
                   <ion-item lines="none">
                     <ion-thumbnail slot="start" class="mobile-only">
-                      <Image src="https://cdn.shopify.com/s/files/1/0069/7384/9727/products/test-track.jpg?v=1626255137" />
+                      <Image :src="getProduct(product.productId).mainImageUrl" />
                     </ion-thumbnail>
                     <ion-label>
-                      <p>Brand</p>
-                      Virtual Name
-                      <p>{{ $t("Color") }} : color</p>
-                      <p>{{ $t("Size") }}: size</p>
+                      <p>{{ getProduct(product.productId).brandName }}</p>
```

Comment:
I think we should not use getProduct() getter here as product has all these information

Reply:
removed getter from view page.